presto-hive/src/main/java/com/facebook/presto/hive/CoercionPolicy.java

```diff
@@ -1,19 +0,0 @@
-/*
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package com.facebook.presto.hive;
-
-public interface CoercionPolicy
```

Comment:
What's the purpose of this commit? 

The reason we have such an interface is to be able to override the coercion policy in our internal connector that's based on the Hive connector.

Reply:
Oh. I did not know about the other implementation. I think (not sure exactly as it was long time ago ;) ) that I assumed that an interface is not needed for internal class with just a single implementation.  I will revert this one.